codalab/model/bundle_model.py

```diff
@@ -2910,18 +2961,71 @@ def create_bundle_store(self, user, name, storage_type, storage_format, url, aut
             connection.execute(cl_bundle_store.insert().values(bundle_store_value))
         return uuid
 
-    def save_oauth2_auth_code(self, grant):
-        with self.engine.begin() as connection:
-            result = connection.execute(oauth2_auth_code.insert().values(grant.columns))
-            grant.id = result.lastrowid
-        return grant
-
-    def delete_oauth2_auth_code(self, auth_code_id):
+    def update_bundle_store(self, user, uuid, update_fields):
+        """
+        update a bundle store
+        """
         with self.engine.begin() as connection:
             connection.execute(
-                oauth2_auth_code.delete().where(oauth2_auth_code.c.id == auth_code_id)
+                cl_bundle_store.update()
+                .where(and_(cl_bundle_store.c.uuid == uuid, cl_bundle_store.c.owner_id == user,))
+                .values(update_fields)
             )
 
+    def get_bundle_store(self, user, uuid):
+        """
+        Return the bundle store corresponding to the specified uuid.
```

Comment:
Can you update all docstrings to document argument names under an  "Arguments:" heading, such as in https://github.com/codalab/codalab-worksheets/blob/d83b8b81e735a6789feda92255654e35311c68ca/codalab/worker/worker.py#L513?